.github/workflows/ci.yml

```diff
@@ -90,30 +154,55 @@ jobs:
       with:
         flag-name: '${{ matrix.db-backend }}: ${{ matrix.python-version }}'
         parallel: true
-    # end-to-end tests
-    - uses: actions/setup-node@v4
+
+  test-e2e:
+    name: "End-to-end Test (Python: ${{ matrix.python-version }}, DB: ${{ matrix.db-backend }})"
+    needs: build-wheel
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        python-version: ['3.12']
+        db-backend: [postgres]
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v5
       with:
-        node-version: 18
-        cache: npm
-      if: matrix.python-version == '3.12' && matrix.db-backend == 'postgres'
+        python-version: ${{ matrix.python-version }}
+        cache: pip
+    - name: Download wheel
+      uses: actions/download-artifact@v4
+      with:
+        name: wheel
+        path: dist
+    - name: Install Dependencies
+      run: |
+        sudo apt-get update && sudo apt install --yes pandoc texlive-xetex librsvg2-bin
+        python -m pip install --upgrade pip
+    - name: Install rdmo[postgres] from wheel and start postgresql
+      run: |
+        python -m pip install "$(ls dist/*.whl)[ci,postgres]"
+        sudo systemctl start postgresql.service
+        pg_isready
+        sudo -u postgres psql --command="CREATE USER postgres_user PASSWORD 'postgres_password' CREATEDB"
+    - name: Prepare Env
+      run: |
+        cp -r testing/media testing/media_root && mkdir testing/log
     - name: Install e2e tests dependencies
       run: |
-        npm install
-        npm run build:prod
-        playwright install chromium
-      if: matrix.python-version == '3.12' && matrix.db-backend == 'postgres'
+        playwright install --with-deps chromium
     - run: mkdir screenshots
+    - name: Collect static files into static root (only required if rdmo is installed from wheel)
+      run: python testing/manage.py collectstatic --noinput
```

Comment:
I could not make the live_server work with the install from the wheel without running collectstatic first.

The webpack bundle is part of the wheel alright, but it is not available without running collectstatic. Do you have any idea how to fix this?

This is why I needed a manage.py here.

Reply:
Since you have already tried with `DEBUG=True` etc., we could further investigate why the static folders are not used but I think that with `testing/manage.py` is the way for now..